server.py

```diff
@@ -175,22 +181,65 @@ def statistics():
     )
 
 
+@app.get("/addQuestion")
+@login_required
+def addQuestion():
+    return bottle.template("addQuestion.html")
+
+
+@app.post("/questionInput")
+@login_required
+def questionInput():
+    userid = Session.get(Session.token == bottle.request.get_cookie("s_id")).user
+    time = datetime.datetime.now()
+    uploaded_question = bottle.request.files.get("question").file.read()
+    uploaded_answer = bottle.request.files.get("answer").file.read()
+    uploaded_statement = bottle.request.forms.get("statement")
+    try:
+        Question.create(
+            test_case_input=uploaded_question,
+            test_case_output=uploaded_answer,
+            question_statement=uploaded_statement,
+            author=userid,
+            created_date_time=time,
+        )
+    except:
```

Comment:
an empty except is not good

```suggestion
    except Exception as e:
```